README.md

```diff
@@ -7,14 +7,14 @@ A tiny state manager for **React**, **React Native**, **Preact**, **Vue**,
 **Svelte**, **Solid**, **Lit**, **Angular**, and vanilla JS.
 It uses **many atomic stores** and direct manipulation.
 
-* **Small.** Between 286 and 818 bytes (minified and brotlied).
+- **Small.** Between 287 and 818 bytes (minified and brotlied).
```

Comment:
Can you revert code style changes in docs?

Reply:
Yes. Was careful to avoid it until I did a search and replace for 286 -> 287 :)